### ISSUE TITLE
Training Related

### ISSUE THREAD

**Issue by xinkez** [Ereboas/MagiCodec#1]
Hi,

Thanks for sharing your great work. I have a few questions about training in your paper.

1. In section 3.1, " a Transformer with a left-context window of 32 and a right-context window of 2 to enhance reconstruction quality while preserving streaming properties.", **Here you choose the right-context window of 2, have you experimented other values? In other words, does the model performance increase with larger value?**

2. In section 3.2.2, "For the input frame X, we independently sample per-frame masks mt ∼ Bernoulli(p). If mt = 1, the original frame is fully replaced by i.i.d. Gaussian noise ϵt ∼ N (0, σ2I);". **what is the value of σ2 used in your training?**

 Thanks in advance.

**Comment by auzxb** [comment]
1.  Increasing the value did not improve the model's performance; values of 1–2 seem to be sufficient, as confirmed by our results.
2. σ² = 1

The paper is already sufficiently detailed. The training code will not be released, but you are welcome to conduct further experiments based on this work.

**Comment by xinkez** [comment]
thanks for your response.

is gaussian noise injection used in all 3 training stages, or only used in stage 3?

**Comment by auzxb** [comment]
only stage1 for training the representation
